styles/index.css

```diff
@@ -0,0 +1,671 @@
+  position: absolute;
+  background-color: var(--color-theme);
+  opacity: 0.6;
+}
+
+.past-summits .summit .year {
+  font-size: 2rem;
+  font-family: var(--font-open);
+  color: var(--color-main);
+}
+
+.past-summits .summit .event {
+  color: white;
+}
+
+ul {
+  padding-left: 0;
+  list-style-type: none;
+}
+
+.program {
+  display: flex;
+  flex-flow: column;
+  width: 100%;
+  height: auto;
+  margin-top: 0;
+  padding-bottom: 30px;
+  background-image: url("../img/pattern.png");
+}
+
+.program .program-title {
+  margin: 40px auto 30px auto;
+  text-align: center;
+  color: white;
+  font-family: var(--font-lato);
+}
+
+.program ul {
+  display: flex;
+  flex-flow: column;
+  padding-left: 0;
+  list-style-type: none;
+}
+
+.program ul li:hover {
+  border: solid 1px var(--color-main);
+  box-sizing: border-box;
+}
+
+.program ul li .icon {
+  width: 2rem;
+  height: 2rem;
+  margin: 20px;
+}
+
+.program h4 {
+  width: 115px;
+  color: var(--color-theme);
+}
+
+.program .button-join {
+  display: inline-block;
+  text-align: center;
+  color: #fff;
+  width: 70%;
+  margin: 40px auto;
+  padding: 30px 20px;
+  background-color: var(--color-theme);
+}
+
+.program .button-join:hover {
+  background-color: var(--color-main);
+  border: 2px solid var(--color-theme);
+  color: var(--color-theme);
+}
+
+.speakers {
+  width: 100%;
+}
+
+.speakers ul {
+  width: 100%;
+}
+
+.speakers ul li:hover {
+  transform: scale(1.1);
+  background-color: #f5eded;
+  -webkit-transform: scale(1.1);
+  -moz-transform: scale(1.1);
+  -ms-transform: scale(1.1);
+  -o-transform: scale(1.1);
+}
+
+.partners img {
+  width: 40px;
+  height: 40px;
+}
+
+.footer img {
+  width: 120px;
+}
+
+.speakers ul li img {
+  max-height: 150px;
+  margin-right: 15px;
+}
+
+.speakers ul li .name {
+  font-family: var(--font-open);
+  margin: 0;
+  padding-top: 5px;
+}
+
+.speakers ul li .field {
+  font-family: var(--font-lato);
+  font-style: italic;
+  color: var(--color-theme);
+  margin: 0;
+  padding-top: 10px;
+}
+
+.speakers ul li .line-s {
+  width: 30px;
+  height: 2px;
+  background-color: var(--color-main);
+  margin: 15px 0;
+}
+
+.speakers ul li .description {
+  font-size: 0.8rem;
+  color: var(--color-dark);
+  margin: 0;
+}
+
+.speakers .more-speakers {
+  cursor: pointer;
+  text-align: center;
+  border: 1px solid var(--color-main);
+  padding: 20px 0;
+  margin: 50px 15px;
+}
+
+.speakers .more-speakers:hover {
+  background-color: var(--color-main);
+}
+
+.speakers .more-speakers i {
+  color: var(--color-theme);
+  margin-left: 15px;
+}
+
+.partners {
+  margin: 80px auto 30px auto;
+  padding-bottom: 40px;
+  text-align: center;
+  background-color: var(--color-dark);
+  font-family: var(--font-lato);
+}
+
+.partners ul li:hover {
+  transform: scale(1.2);
+  -webkit-transform: scale(1.2);
+  -moz-transform: scale(1.2);
+  -ms-transform: scale(1.2);
+  -o-transform: scale(1.2);
+}
+
+.footer {
+  padding: 0 30px 40px 30px;
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```